Homework3/Homework3/Program.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.IO;
+
+namespace Homework3
+{
+    /// <summary>
+    /// Program allows users to log in or register new login credentials.
+    /// </summary>
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string filePath = @"Users.txt";
```

Comment:
@".." is only needed when you work with `\` (so you don't have to write `\\`). @ is not necessary in this cae.